src/models/container.ts

```diff
@@ -29,13 +30,18 @@ export const Container = ({initialDiagram, initialItemList, initialSharedModel}:
       //    the tile views
       // If we support tiles having customized views of shared models then this will
       // need to become more complex.
-      for (const tree of Object.entries(trees)) {
-        if (tree[0] === sourceTreeId) continue;
-  
+      const applyPromises = Object.entries(trees).map(tree => {
```

Comment:
You could make the code a bit more readable with:
```suggestion
      const applyPromises = Object.entries(trees).map(([treeId, treeValue]) => {
```
there are probably better names than `treeId` and `treeValue`, but I leave that to your discretion.

Reply:
Thanks! Yes I had tried this before but got confusing errors because I forgot to wrap the param in `(...)`